CMSgov/onemac
# Update titles for required details pages

Story: https://qmacbis.atlassian.net/browse/OY2-22187

### Details

Updates the titles for:
- Appendix K details
- CHIP SPA details
- Medicaid SPA details
- Initial Waiver details

### Changes

Each file has a corresponding component that passes props into a general Detail component for rendering. These props are overridable -- each page's `detailHeader` prop was overridden with the required value.

### Implementation Notes

- Opted against a data change in favor of overriding through props to ensure no side effects from changing data

### Test Plan

1. Login using a Dev login
2. Locate one of each type of details page
3. Ensure the title match the AC of the ticket
